projects/chat-lib/src/lib/chat-lib.service.ts

```diff
@@ -9,14 +9,16 @@ import { Injectable } from '@angular/core';
 })
 export class ChatLibService {
 
-  baseUrl: string = 'https://chatapi.diksha.gov.in/bot';
+  baseUrl: string ;
```

Comment:
@ajinkyaghodake123 - this separate variable not needed, You can use strait variable that is coming from, Chat Lib Component

Reply:
done